.github/workflows/manual-nixos.yml

```diff
@@ -1,9 +1,8 @@
 name: "Build NixOS manual"
 
-permissions: read-all
-
 on:
-  pull_request_target:
+  push:
+  pull_request:
```

Comment:
Is this right? It seems weird that an empty body for an `on` value would fall through to the next, but I'd believe it if you said so :)

Reply:
Yes. This is correct syntax.

Reply:
Currently this change will start building the manuals on future release branches, would need to be updated to keep the original behaviour of only building master.

```
push:
   branches:
      - master
```